packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -52,8 +52,8 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
       const transform = async(bindings: Bindings, next: any, push: (result: IAnnotatedBinding) => void):
       Promise<void> => {
         try {
-          const result = await evaluator.evaluate(bindings);
-          push({ bindings, result });
+          const result = await evaluator.evaluateAsInternal(bindings);
```

Comment:
If `toRDF` is undefined, isn't this a problem in sparqlee?

I guess we should be able to keep using `evaluate`.

Reply:
Perfect! :-)

I just published sparqlee 2.1.0. Could you update the PR to use that version? Will merge right away after that.

Reply:
Thank you! I have just done it.